extension/extension/inject.js

```diff
@@ -0,0 +1,13 @@
+let script = document.createElement('script')
```

Comment:
Stick a header on this file explaining what role it plays and when you expect it to be called. please.

Reply:
I've added this